frontend/src/api/profile/getToken.tsx

```diff
@@ -0,0 +1,7 @@
+import { useMutation } from 'react-query';
+import putUserToken from '../restApi';
```

Comment:
Filter imports

Reply:
Remove please this file. It's an auth module, not profile!

Reply:
Removed and created correct folder/file